unit_test/test.cpp

```diff
@@ -14,8 +14,8 @@ PFHub1a Available here (ORNL internal): https://code.ornl.gov/71d/phase-field-ex
 selected
 */
 TEST(PFHub1a, Initialization) {
-    PFHub1aBenchmark simulation(96, 500);
-    simulation.timestep(0); // trigger initialization
+    PFHub1aBenchmark simulation(96, .5);
+    simulation.run(0); // trigger initialization
```

Comment:
I think it may be more clear to just call `initialize()`, but either way is fine

Reply:
I agree with @streeve -- calling `initialize()` would be clearer than doing a zero-step solve.